src/rpc-request.js

```diff
@@ -42,9 +41,18 @@ var rpcRequest = function (path, body, accessToken, selectUser) {
       body = null;
     }
 
-    apiRequest = request.post(BASE_URL + path)
-      .type('application/json')
-      .set('Authorization', 'Bearer ' + accessToken);
+    apiRequest = request.post(getBaseURL(host) + path)
+      .type('application/json');
+
+    switch (auth) {
+      case 'user':
```

Comment:
Sorry, I didn't realize team routes were supported at all. I now see `team-routes.js`, which apparently didn't get regenerated. (I used `grep` to make sure that everything in my diff used either "user" or "noauth" auth, but the failure here was that my diff didn't actually regenerate everything.)

No tests from `npm test` failed... odd. I'll investigate.